cv32e40s/fv/q.tcl

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Silicon Labs, Inc.
```

Comment:
`q.tcl` is a very short filename.  I assume this is a control file for Questa Formal.  If that is the case, then perhaps a more descriptive filename such as `questa_formal.tcl` would better convey this file's intent.

Reply:
I made new names for both the tcl files and the make targets.